src/plugins/embeddable/public/lib/embeddables/compatibility/legacy_embeddable_to_api.ts

```diff
@@ -189,20 +196,23 @@ export const legacyEmbeddableToApi = (
     embeddable.updateInput({ timeRange: nextTimeRange });
 
   const filters$: BehaviorSubject<Filter[] | undefined> = new BehaviorSubject<Filter[] | undefined>(
-    undefined
+    isFilterableEmbeddable(embeddable) ? embeddable.getFilters() : undefined
```

Comment:
Needed to set filters$ and query$ to values. Otherwise, newSession$ fired because `embeddable.untilInitializationFinished().then` called `filters$.next` and `query$.next`